docs/architecture/adr-050-sign-mode-textual-annex1.md

```diff
@@ -320,6 +320,10 @@ TestData object
 > Signer: cosmos1abc
 ```
 
+### bool
+
+Boolean values are rendered as `YES` or `NO`.
```

Comment:
What about this?

```suggestion
Boolean values are rendered as `True` or `False`.
```

I wonder if yes/no is a good mapping for the general bool type. Also avoiding SCREEM CASE is friendlier IMO.